.changeset/update-flags-stale-cli.md

```
@@ -0,0 +1,13 @@
+---
+"@fission-ai/openspec": patch
+---
+
+`openspec update` now tells you when your globally installed CLI is behind the published one. Instruction files are generated by the installed CLI, so a stale global install would report `✓ All 1 tool(s) up to date (v1.6.0)` while the workflows added in newer releases were never written — the fix for that is `npm install -g @fission-ai/openspec@latest` first, which the command now says out loud:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)